src/selectors/profile.js

```diff
@@ -527,6 +539,16 @@ export const getLocalTrackName = (
     'Could not find the track names from the given pid'
   )[trackIndex];
 
+export const getLocalTrackTooltip = (
+  state: State,
+  pid: Pid,
+  trackIndex: TrackIndex
+): ?string =>
+  ensureExists(
+    getLocalTrackTooltipsByPid(state).get(pid),
+    'Could not find the track names from the given pid'
+  )[trackIndex];
+
```

Comment:
I think you don't use these 2 selectors now, so I'd suggest to remove all the code related to tooltips for now.
Happy to look at something related in a future PR if you want, but let's clean up this PR for now.